src/main/java/org/dormitory/autobotsoccub/command/txtcommand/HelloCommand.java

```diff
@@ -47,6 +47,6 @@ private String userNameFromUpdate(Update update) {
     }
 
     private String toUserName(User user) {
-        return user.getFirstName() + " " + user.getLastName();
+        return user.getFirstName() + (user.getLastName() != null ? " " + user.getLastName() : "");
```

Comment:
[future_notes] This should be moved to some kind of UserOperations service/util class, encapsulate user-specific logic there. Let's leave it as is for now.

Reply:
Moved to the UserUtils class